livekit-plugins/livekit-plugins-azure/livekit/plugins/azure/stt.py

```diff
@@ -330,6 +333,7 @@ def _create_speech_recognizer(
             speechsdk.enums.PropertyId.Speech_SegmentationStrategy,
             str(config.segmentation_strategy),
         )
+    speech_config.set_profanity(config.profanity)
```

Comment:
more defensive to only set it when it's not None. sometimes sdks don't like none values

```suggestion
    if config.profanity is not None:
        speech_config.set_profanity(config.profanity)
```